### DISCUSSION TITLE
Stage 4: System integration & composite testing.

### DISCUSSION THREAD

**Discussion by sanket1211996** [airavata-courses/SuperNova#61] [Ideas]
I have opened up the Stage 4 of the project in which we will discuss on integration work and testing status related to it.

**Comment by sanket1211996** [comment]
I have started the integration of weather api and user api services. I have raised the PR #57  for UI changes regarding the issue #62. Do review and merge.

**Reply by sumedhsalvi** [reply]
PR reviewed and merged

**Comment by sanket1211996** [comment]
I have created Issue Gateway Integration Test #65 and raised PR #64   for some required changes found during test. @sumedhsalvi  do verify and merge.

**Comment by sanket1211996** [comment]
I have create PR #66, with changes in UI for integration with gateway. Do verify and merge.

**Reply by sumedhsalvi** [reply]
PR reviewed and merged

**Comment by sanket1211996** [comment]
I have create #69 for the issue #62 . Please do review and merge.

**Reply by sumedhsalvi** [reply]
PR reviewed and merged

**Comment by sanket1211996** [comment]
I have create the PR #70 for the issue #62 . Please do review and merge.

**Reply by sumedhsalvi** [reply]
PR reviewed and merged

**Comment by sumedhsalvi** [comment]
I have created PR #72 for issue #71.
Please review and merge.

**Comment by sanket1211996** [comment]
I have created weather cache integration issue #73
